install.sh

```diff
@@ -258,14 +261,16 @@ __add_bro_to_path() {
 __install_mongodb() {
 	case "$_OS" in
 		Ubuntu)
-			__add_deb_repo "deb [ arch=$(dpkg --print-architecture) ] http://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/3.4 multiverse" \
-				"MongoDB" \
-				"https://www.mongodb.org/static/pgp/server-3.4.asc"
+			__add_deb_repo "deb [ arch=$(dpkg --print-architecture) ] http://repo.mongodb.org/apt/ubuntu $(lsb_release -cs)/mongodb-org/${_MONGO_VERSION} multiverse" \
+				"mongodb-org-${_MONGO_VERSION}" \
+				"https://www.mongodb.org/static/pgp/server-${_MONGO_VERSION}.asc"
 			;;
 		CentOS)
-			if [ ! -s /etc/yum.repos.d/mongodb-org-3.4.repo ]; then
-				echo -e '[mongodb-org-3.4]\nname=MongoDB Repository\nbaseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.4/x86_64/\ngpgcheck=1\nenabled=1\ngpgkey=https://www.mongodb.org/static/pgp/server-3.4.asc' | tee /etc/yum.repos.d/mongodb-org-3.4.repo > /dev/null
+			# __add_rpm_repo "https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/${_MONGO_VERSION}/x86_64/"
+			if [ ! -s /etc/yum.repos.d/mongodb-org-${_MONGO_VERSION}.repo ]; then
+				echo -e '[mongodb-org-${_MONGO_VERSION}]\nname=MongoDB Repository\nbaseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/${_MONGO_VERSION}/x86_64/\ngpgcheck=1\nenabled=1\ngpgkey=https://www.mongodb.org/static/pgp/server-${_MONGO_VERSION}.asc' | tee /etc/yum.repos.d/mongodb-org-${_MONGO_VERSION}.repo > /dev/null
 			fi
+			__freshen_packages
```

Comment:
If we used the `__add_rpm_repo` function we wouldn't need this because that function does this. I'm not actually sure why Mongo installed correctly without this before on CentOS. Ubuntu requires an `apt-get update` after a repo is added, but maybe CentOS does not?